### ISSUE TITLE
Allow to specify quantile values for the whiskers of boxplots

### ISSUE THREAD

**Issue by cristobaltapia** [JuliaPlots/StatsPlots.jl#449]
Hi, I am a rather new user of StatsPlots.jl and Julia. Thanks a lot for this package! I was doing some boxplots where I would like to mark the whiskers at the 5% and 95% quantile values. However, I could not find a way to do so. Reading through the implementation it looks like it is only possible to define the `range` value. Would it be possible to allow to overload `range` so that it also accepts a tuple of values e.g. `(0.05, 0.95)` and use it to draw the whiskers instead if `IQR * range`? Or maybe accept a tuple of any two arbitrary values that should be used for the whiskers?

Thanks!

**Comment by BeastyBlacksmith** [comment]
Is https://github.com/JuliaPlots/StatsPlots.jl/pull/440 implementing what you are looking for_

**Comment by cristobaltapia** [comment]
@BeastyBlacksmith yes! That is exactly what I was looking for. Hopefully it will get merged soon.

**Comment by cristobaltapia** [comment]
I just tested this PR but it is not quite what I was looking for. What #440 implements is to modify the range of the box and add additional lines within the box, but what I would like to have is the possibility to define a custom value (or percentile) for the _whiskers_, i.e. the lines extending beyond the box.

**Comment by HenriDeh** [comment]
I closed because I got no reaction to the PR and I'm not using StatPlots anymore. I can reopen if you like.

**Comment by cristobaltapia** [comment]
I actually implemented a solution myself in the meantime. I like #401, however I think that it would be better to allow to specify the upper and lower quantiles for the whiskers independently (as done in [matplotlib](https://matplotlib.org/stable/api/_as_gen/matplotlib.pyplot.boxplot.html)). However, to be honest, I have never used _non-symmetric_ quantiles for this, so I don't know if this is really necessary. But since it is so defined in matplotlib, then there might be a use for it.
